### PR TITLE
Added finnish language

### PR DESCRIPTION
**Pull request by karkkijokeri**

### CODE REVIEW

**Comment by karkkijokeri** [comment]
I translated it to Finnish xD

File: language/lang/fi-FI.js [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+  },
+  errors: {
+    nothingPlaying: musicErrors.nothingPlaying,
+    sameChannel: musicErrors.sameChannel
+  }
+}
+export const play = {
+  author: 'Lisätty jonoon.',
+  fields: {
+    amount: { name: 'Määrä', value: (amount) => `${amount} songs` },
```
Comment (MeridianGH) [inline language/lang/fi-FI.js]: Translation missing: songs

File: language/lang/fi-FI.js [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+  title: 'Kutsu SuitBot',
+  description: 'Klikkaa tästä linkistä kutsuaksesi SuitBotin palvelimellesi!'
+}
+export const language = {
+  author: 'Kieli.',
+  title: 'Vaihda kieltä',
+  description: (langCode) => `Kieli asetettiin \`${langCode}\`.`
+}
+export const ping = {
+  author: 'Viive',
```
Comment (MeridianGH) [inline language/lang/fi-FI.js]: Can you add a dot here? Viive.

File: language/lang/fi-FI.js [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+  errors: {
+    nothingPlaying: musicErrors.nothingPlaying,
+    sameChannel: musicErrors.sameChannel
+  }
+}
+export const lyrics = {
+  author: 'Sanat.',
+  other: {
+    repeatModes,
+    genius: 'Provided by genius.com',
```
Comment (MeridianGH) [inline language/lang/fi-FI.js]: Could you translate this as well?

**Review by MeridianGH — CHANGES_REQUESTED** [review]
Nice! Thank you!
Can you do some translations you seem to have missed?
I made comments on the lines in question :)

File: language/lang/fi-FI.js (hunk 2 above)
Comment (karkkijokeri) [reply language/lang/fi-FI.js]: yes

**Comment by karkkijokeri** [comment]
I fixed everything you asked

**Comment by MeridianGH** [comment]
Awesome, thanks a lot!

**Pull request merged**